bundles/org.openhab.core.compat1x/src/main/java/org/openhab/io/net/exec/ExecUtil.java

```diff
@@ -137,7 +137,7 @@ public static String executeCommandLineAndWaitResponse(String commandLine, int t
                 logger.debug("exit code '{}', result '{}'", exitCode, retval);
             }
         } catch (InterruptedException e) {
-            logger.error("Timeout occured when executing commandLine '" + commandLine + "'", e);
+            logger.error("Timeout occurred when executing commandLine '" + commandLine + "'", e);
```

Comment:
I guess you should have also changed it to parametrized logging at the same time :-)